xml/compute.xml

```diff
@@ -20,10 +20,10 @@
     instances of the same libraries might exist, differing in version, build
     configuration, compiler, and MPI implementation. To manage these
     dependencies, you can use an environment module system. Most &hpca;
-    libraries provided with &slehpc; are built with support for environment
+    libraries provided with the &hpca; Module for &sles; are built with support for environment
```

Comment:
```suggestion
    libraries provided with the &hpca; module for &sles; are built with support for environment
```

Reply:
Best double-check with @dariavladykina but I think we use lower case spelling for 'module'. At least this seems to be the default spelling in the 'Modules & Extensions Quick Start.'

Reply:
I would keep it capitalized in this guide because it is consistent throughout and, I see it as a 'product' name. The great majority of 'module' instances in `doc-sle` and `doc-hpc` is capitalized, even though the use is not consistent.

Reply:
I'm with @taroth21 on this one. We usually lowercase module. Capitals are only used if it's a screen, term or literal. Most of the capital results you get when grep'ing through doc-sle are from SUSEConnect in screens.

Reply:
Happy with lowercase unless it's a screen/literal.